roles/openshift_hosted/defaults/main.yml

```diff
@@ -116,3 +116,5 @@ openshift_hosted_docker_registry_insecure_default: "{{ openshift_docker_hosted_r
 openshift_hosted_docker_registry_insecure: "{{ openshift_hosted_docker_registry_insecure_default }}"
 
 openshift_hosted_registry_storage_azure_blob_realm: core.windows.net
+
+l_default_replicas: 1
```

Comment:
Let's do a little housekeeping and just set
openshift_hosted_registry_replicas: 1

in defaults and remove all references to l_default_replicas, it's no longer needed if we're removing the dynamic counting of infra hosts.